### ISSUE TITLE
Class selection screen

### ISSUE THREAD

**Issue by TrustyPatches** [open-holloway-makers/dehiscent-rpg#8]
Before the main loop of the game begins, the player should have the ability to choose from a selection of classes.

**Comment by vasuagrawal84** [comment]
What kind of classes were you thinking? Something like elf, drawf, ninja etc? 
Would these replace the current Wanderer or are we planning to keep that one?

**Comment by getachi** [comment]
Elf and dwarf classes? It almost sounds like you have your life together and don't spend hours on end playing video games. _nerd_.

But yeah on that note having races could be an addition on top of classes but lets implement that first. I can't speak for Patches but classes like ninja, warrior, mage etc sounds like the right idea

**Comment by TrustyPatches** [comment]
Yeah what getachi said, wanderer should remain as a kind of prototypical class. Also elves and dwarves are lame ;) If we were to add races it should probably be after they've been naturally introduced by people in their cells, so that that race already has a presence and history in the world.

**Comment by vasuagrawal84** [comment]
I had a go at making the class selection screen with a sample Ninja class added. It's on my repo. Give it a go and tell me what you guys think.
